cgi-bin/DW/Captcha.pm

```diff
@@ -209,10 +209,14 @@ sub validate {
 
     my $err_ref = $opts{err_ref};
 
-    if ( $self->challenge ) {
-        return 1 if $self->_validate;
+                    # captcha type, page captcha appeared on
+    my $stat_tags = [ (ref $self)->name, "page:" . $self->page ];
```

Comment:
What is `name` and `page` here? We can't insert completely arbitrary tags, they have to fit some requirements -- basically `[a-z0-9_-:.]` or something.